SPECS/moreutils/moreutils.spec

```diff
@@ -0,0 +1,404 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+## START: Set by rpmautospec
+## (rpmautospec version 0.3.5)
+## RPMAUTOSPEC: autorelease, autochangelog
+%define autorelease(e:s:pb:n) %{?-p:0.}%{lua:
```

Comment:
Do we use this macro anywhere ?
if not we should drop it.

Reply:
No, this macro is not used anywhere. dropping it